common/va_display_drm.c

```diff
@@ -81,7 +82,9 @@ va_open_display_drm(void)
             close(drm_fd);
             continue;
         }
-        if (!strncmp(version->name, "vgem", 4)) {
```

Comment:
Do you mind adding a small comment:
On normal Linux platforms we do not want vgem. Yet WSL uses vgem, while also providing a fallback VA driver (see libva for more)